### DISCUSSION TITLE
Admin Front-End translation lost since update

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#673] [General]
Hello I've update to Wordpress 6..7 and EME 2.5.16.

The administration Front End of EME stay in English, trie to change the language of Wordress in nl_BE.
Wordpress Admin FE change into NL except EME.

Anyone have also this issue ?

**Comment by liedekef** [comment]
it seems that wordpress changed the logic of translation loading (again). I'm filtering/calling the acceptable WP functions in time, I've traced all calls and it should work, but it doesn't (go figure: a new WP bug).
Anyway, I've rewritten the logic to no longer call load_plugin_textdomain but load_textdomain directly. This seems less buggy, see this changeset:

https://github.com/liedekef/events-made-easy/commit/6b5dc0f532393dfbcb861d80350cf12a9728e54f

**Comment by MichaelWybo** [comment]
Solved. Thanks
